docs/guide/Titanium_SDK/Titanium_SDK_Release_Notes/Titanium_SDK_Release_Notes_12.x/Titanium_SDK_12.6.4.GA_Release_Note.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Titanium SDK 12.6.4.GA - 04 April 2025
+weight: '19'
```

Comment:
```suggestion
weight: '34'
```

Can you check https://github.com/tidev/titanium-docs/pull/239 first and then make this 34 so it will be in the correct order in the left sidebar